### DISCUSSION TITLE
Discuss distros

### DISCUSSION THREAD

**Discussion by lagergren** [xtclang/xvm#93] [Ideas]
Tools: jreleaser
Distro format 
brew/apt/msi etc
Using Gradle Native plugin to cross compile Native launches.

**Comment by cpurdy** [comment]
We have a way of doing brew installs (i.e. mainly for macOS), and Windows installs (we still need to add "win get" support for Windows, though).

So Linux is the big "missing link", and unfortunately that means supporting several popular distribution formats, because Linux of course. I'm assuming that we need apt (.deb) and rpm (covers yum?) as a minimum, and we already have .zip and .tar.g for manual installs.
